server/controllers/usage-controller.js

```diff
@@ -96,4 +96,82 @@ exports.getBookingStats = async (req, res) => {
         });
     }
   };
-  
\ No newline at end of file
+  
+  exports.getSystemStats = async (req,res)=>{
```

Comment:
Only Manager should be able to access this data.

Reply:
ok, now it sent an error message if anyother role try to access to the queries